packages/s3-replicator/README.md

```diff
@@ -0,0 +1,21 @@
+# S3 Replicator
+
+This package includes a simple lambda functions and associated permissions to replicate create-object events from one S3 bucket to another.
+It was developed to enable same-region cross-account object replication.
+
+## Configuration
+
+Configuration variables are shown in `cfg.tfvars.sample`, and are explained below:
+
+```
+prefix               = "myprefix"             # prefix to name created replicator resources
+aws_profile          = "ngap"                 # AWS profile used for deployment
+permissions_boundary = "NGAPShRoleBoundary"   # IAM permissions boundary
+vpc_id               = "vpc-123456"           # VPC ID (for Lambda)
+subnet_ids           = ["subnet-123456"]      # Subnet IDs (for Lambda)
+security_groups      = ["sg-123456"]          # Security Group IDs (for Lambda)
+source_bucket        = "source-bucket-name"   # Source S3 bucket
+source_prefix        = "source-prefix"        # Source object prefix e.g. 'path/to/filedir'
+target_bucket        = "target-bucket-name"   # Target S3 bucket e.g. 'mybucket'
+target_prefix        = "target-prefix"        # Target object prefix path
+```
```

Comment:
Probably need some basic instructions on how to deploy, or point to basic instructions.

Reply:
I'm unable to deploy and I'm sure I've got something configured wrong.  getting errors about my user not having permissions

```Error: Error creating IAM Role mhs3_replicator_lambda_role: AccessDenied: User: arn:aws:iam::123456789012:user/CumulusSandboxAppDeveloper-mhsavoie-137 is not authorized to perform: iam:CreateRole on resource: arn:aws:iam::123456789012:role/mhs3_replicator_lambda_role ```

Reply:
Do you have AWS credentials in your environment variables? They will override your `aws_profile`  in the `.tfvars` file, so make sure they aren’t for a different account or something. I’ll add a note about what command to deploy with.